### PR TITLE
Fix spack develop title and texts

### LINKED ISSUE #828: Change "Build an ACCESS model" to "Modify an ACCESS model's source code"
## Description

During user testing of https://access-hive.org.au/models/run-a-model/build_a_model/ , Kial suggested substituting `Build` with `Recompile`.

I think "Recompile", "Rebuild" or "Modify" would be more descriptive than "Build".

### LINKED ISSUE #832: spack develop command is wrong
In the [Build an ACCESS model](https://github.com/ACCESS-NRI/access-hive.org.au/blob/development/docs/models/run-a-model/build_a_model.md) page, the spack develop command is wrong:

[This line](https://github.com/ACCESS-NRI/access-hive.org.au/blob/51278bfab8b1ec07dcfb752428ff9cbba7b89c8d/docs/models/run-a-model/build_a_model.md?plain=1#L317):

`spack develop mom5@development_version --path /path/to/mom5/new/source/code`

should be 

`spack develop --path /path/to/mom5/new/source/code mom5@development_version `

### PR DESCRIPTION
**Pull request by atteggiani**
- [x] Fixed '--path' option order to 'spack develop' command. Fixes #832.
- [x] Changed 'Build an ACCESS model' to 'Modify an ACCESS model's source code' + minor adjustments to the texts. Fixes #828.

### CODE REVIEW

**Comment by harshula** [comment]
Hi @atteggiani , Do any of these need to be updated?

```
models/run-a-model/index.md
33:            <img class="img-contain with-padding white-background" src="/assets/model-config-logos/globe_visualisation/access_om_globe_visualisation.png" alt="Build a model">

models/run-a-model/run-access-esm.md
645:To customise {{ model }}'s build (for example to run {{ model }} with changes in the source code of one of its component), refer to [Build {{ model }}](/models/run-a-model/build_a_model#{{model|lower}}).

models/run-a-model/run-access-om.md
568:To customise {{ model }}'s build (for example to run {{ model }} with changes in the source code of one of its component), refer to [Build {{ model }}](/models/run-a-model/build_a_model#{{model|lower}}).
```

File: docs/models/run-a-model/build_a_model.md [hunk 1]
```diff
@@ -6,15 +6,14 @@
 {% set spack_setup = "/getting_started/spack" %}
 [gadi]: https://opus.nci.org.au/display/Help/0.+Welcome+to+Gadi#id-0.WelcometoGadi-Overview
 
-# Build an ACCESS model
+# Modify an ACCESS model's source code
 
 ## About
 
-The instructions below outline how to build an ACCESS model using the build-from-source package manager [Spack](https://spack.readthedocs.io).<br>
+The instructions below outline how to build the dependencies of an ACCESS model using the build-from-source package manager [Spack](https://spack.readthedocs.io).<br>
```
Comment (harshula) [inline docs/models/run-a-model/build_a_model.md:13]:
```suggestion
The instructions below outline how to build an ACCESS model, and dependencies, using the build-from-source package manager [Spack](https://spack.readthedocs.io).<br>
```

File: docs/models/run-a-model/build_a_model.md [hunk 2]
```diff
@@ -6,15 +6,14 @@
...
 This build workflow is specifically designed to run on the [National Computating Infrastructure (NCI)](https://nci.org.au/about-us/who-we-are) supercomputer [_Gadi_][gadi].
 
-
-As an example, in the following instructions we will show how to build a modified [mom5 component](/models/model_components/ocean/#mom5) for [ACCESS-ESM1.5](/models/configurations/access-esm/#access-esm15). All other components and packages (i.e., dependencies) will remain the same as the official [ACCESS-ESM1.5 release]({{esm1_5_build_config}}).
+As an example, in the following instructions we will show how to modify [mom5 component](/models/model_components/ocean/#mom5) for [ACCESS-ESM1.5](/models/configurations/access-esm/#access-esm15) and re-compile the ACCESS-ESM1.5 dependencies. All the other components and packages (i.e., dependencies) will remain the same as the official [ACCESS-ESM1.5 release]({{esm1_5_build_config}}).
```
Comment (harshula) [inline docs/models/run-a-model/build_a_model.md:16]:
```suggestion
As an example, in the following instructions we will show how to modify [mom5 component](/models/model_components/ocean/#mom5) for [ACCESS-ESM1.5](/models/configurations/access-esm/#access-esm15) and re-compile the relevant ACCESS-ESM1.5 dependencies. All the other components and packages (i.e., dependencies) will remain the same as the official [ACCESS-ESM1.5 release]({{esm1_5_build_config}}).
```

File: docs/models/run-a-model/build_a_model.md (hunk 1 above)
Comment (atteggiani) [reply docs/models/run-a-model/build_a_model.md:13]: I will modify it to:
```
The instructions below outline how to build an ACCESS model and its dependencies, using ...
```

**Review by harshula — APPROVED** [review]

**Comment by atteggiani** [comment]
Thank you @harshula. 
Merged.

**Pull request merged**